packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -661,12 +661,14 @@ contract Bridge is EssentialContract, IBridge {
             uint256 words = _dataLength / 32 + 1;
             uint256 memoryGasCost = words * 3 + words * words / 512;
 
-            // Need to add the gas cost of accessing the message.to account. This currently
-            // corresponds to 2600 gas if the account is cold, and 100 otherwise.
-
-            // Also need to add the cost of transferring a nonzero msg.value. This cost is currently
-            // 9000, but provides a 2300 gas stipend to the called contract.
-            result_ = gasleft() * 63 < _minGas * 64 + (memoryGasCost + 9300) * 63;
+            // Need to add the cost of the worst case scenario of the external CALL, which include
+            // 700 gas for `CALL`, 2,600 gas for `address_access_cost`, 9,000 for
```

Comment:
there is no 700 gas for the call itself as far as I know, the minimum gas cost for a CALL is 100 gas which is just the warm address access cost.

Reply:
According to https://www.evm.codes/, it does seem the cost of CALL opcode is 100 gas.

Reply:
Updated the code and comment, but keep using 40000

Reply:
> According to https://www.evm.codes/, it does seem the cost of CALL opcode is 100 gas.

I don't think even the 100 gas is a fixed cost for the opcode, that's just the minimum gas cost for the whole opcode:
> static_gas = 0
> dynamic_gas = memory_expansion_cost + code_execution_cost + address_access_cost + positive_value_cost + value_to_empty_account_cost

The static gas is 0, so no fixed cost for the opcode itself. The minimum 100 gas comes from the warm `address_access_cost` I believe.

Reply:
Fixed by our auditor, please take another look.